core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/machine/Server.java

```diff
@@ -36,14 +42,15 @@
     String getProtocol();
 
     /**
-     * Path to access the server.
+     * Url of the server, e.g.&nbsp;http://localhost:8080
```

Comment:
I think you mean "regular space". If I add a regular space and I try to generate the Javadoc I have the following result:

![image](https://cloud.githubusercontent.com/assets/606959/18626032/b4237b6c-7e52-11e6-8797-72cb966a3414.png)

If I use `&nbsp;` instead the sentence is not broken:

![image](https://cloud.githubusercontent.com/assets/606959/18625990/70ca2776-7e52-11e6-962c-376815b3153b.png)